nservicebus/hosting/cloud-services-host/configuration.md

```diff
@@ -1,114 +1,78 @@
 ---
-title: Azure Cloud Services Host Endpoint Configuration
+title: Configuration
 summary: Configuring the endpoint when hosting in Azure Cloud Services
 tags:
 - Azure
 - Cloud
+- Configuration
 ---
 
-## Enabling the Transport
+## Configuring for Cloud Services hosting
 
-When using one of the NServiceBus provided hosting processes, the `UseTransport<T>` should be called on the endpoint configuration. For example using Azure Service Bus Transport
+Cloud Services is a hosting model provided by the Azure cloud, which is specifically designed for hosting large applications. For a detailed description of the cloud service configuration in Azure, see [What is the Cloud Service model and how do I package it?](https://azure.microsoft.com/en-us/documentation/articles/cloud-services-model-and-package/).
 
-snippet:AzureServiceBusTransportWithAzureHost
+When configuring NServiceBus for hosting in Cloud Services, it needs to connect to a specific Azure storage account (for Azure Queues) or a Azure Service Bus namespace. For more information on the connection string formats, refer to [the windows azure connection string formats](http://www.connectionstrings.com/windows-azure/).
 
+## Configuring an endpoint
 
-Example using azure storage queues transport:
+When using the Azure Cloud Services hosting entrypoint to host an endpoint, it should be configured using and implementation of `IConfigureThisEndpoint`.
```

Comment:
> When using the Azure Cloud Services hosting entrypoint to host an endpoint, it should be configured using and implementation of `IConfigureThisEndpoint`.

Not sure if that's not changing the meaning slightly, but how about:

When an endpoint is hosted in the Azure Cloud Services, then it should be configured by implementing the `IConfigureThisEndpoint` interface.